### DISCUSSION TITLE
💪Amazing and great result, thx for this!

### DISCUSSION THREAD

**Discussion by ZeroCool22** [TheLastBen/fast-stable-diffusion#274] [General]
I'm getting great results just training with the **defaults values**.

I tried another repo's and the result weren't this good.

Hope the **Local version** comes soon, and i pray for it to can run on a **1080 TI**.

![7653093f25610de85dc76a59154f49a3](https://user-images.githubusercontent.com/13344308/198680562-fc427c59-ee63-4b24-bfd7-cc49d930abd8.gif)

**Comment by TheLastBen** [comment]
do you have +25GB of RAM ?

**Reply by ZeroCool22** [reply]
> do you have +25GB of RAM ?

Yeah, 32gb of RAM 3000Mhz.

**Reply by TheLastBen** [reply]
good

**Reply by DarkAlchy** [reply]
> good

Are we really going to need that much ram if our vid cards have at least 12 gigs?

**Reply by ZeroCool22** [reply]
> > good
> 
> Are we really going to need that much ram if our vid cards have at least 12 gigs?

Looks like yes...

I see it use **8bit_adam** and i thought with that it makes possible to run in a 1080 TI without need RAM like the **ShivamShrirao** ones does, but seems like not.

Also, don't know how much using DS will slow down the train.

**Reply by DarkAlchy** [reply]
Dang.  Well, my first purchase of 2023 is a 4060.  By 2025 I will have a new computer to retire this 2016 system.  64 gigs of ram (2 sticks) minimum was always going to be my choice.  Maybe by then 128 gigs will be, lol. :/
